test/files/neg/sip23-no-unit-type.check

```diff
@@ -0,0 +1,22 @@
+sip23-no-unit-type.scala:6: error: Illegal literal type (), use Unit instead
+    case _: ()      => "err"
+            ^
+sip23-no-unit-type.scala:7: error: Illegal literal type (), use Unit instead
+    case _: ().type => "err"
+            ^
+sip23-no-unit-type.scala:7: error: '=>' expected but '.' found.
+    case _: ().type => "err"
```

Comment:
it seems odd to me to have this error after the previous error

Reply:
me too, it's due to the ad hoc fix.